arch/src/aarch64/fdt.rs

```diff
@@ -192,32 +189,9 @@ fn create_cpu_nodes(
         fdt.end_node(cpu_node)?;
     }
 
-    // If there is a valid cpu topology config, create the cpu-map node.
-    if (threads_per_core > 0)
```

Comment:
Some explanation of the simplification:

We don't border checking if they are zero. This is done in parameter parsing.